backend/routes/questionRoutes.js

```diff
@@ -3,12 +3,12 @@ const { getAllQuestions, createQuestion, deleteQuestion } = require('../controll
 const router = express.Router();
 
 // Route to get all questions (homepage)
-router.get('/', getAllQuestions);
+router.get('/get-all-questions', getAllQuestions);
```

Comment:
Changing the path would likely mean that our frontend paths and backend paths won't be a 1 to 1 matching (because it is unlikely the frontend path will be "/get-all-questions" but rather "/" or "/homepage" in this example).

I am fine with it, just wanted a heads up.